src/basic/cgroup-util.c

```diff
@@ -292,7 +292,7 @@ int cg_kill(
                 const char *path,
                 int sig,
                 CGroupFlags flags,
-                Set *s,
+                Set *set,
```

Comment:
Maybe better to name `killed_pids` or so? `set` is still ambiguous.

Reply:
Renamed it to `killed_pids` and taken the liberty to also rename 'log_kill' -> 'kill_log' according to the header definition and used argument comments.

Seems that clang-tidy only cares about header files, and not that header / implementation match.

Reply:
Ugh, I'd keep `log_kill` as-is.